doc/source/writers.rst

```diff
@@ -136,3 +136,34 @@ and save them all at once.
     ...                          compute=False)
     >>> results = [res1, res2]
     >>> compute_writer_results(results)
+
+
+Adding text to images
+=====================
+
+Satpy, via `pydecorate`, can add text to images when they're being saved.
+To use this functionality, you must create a dictionary describing the text
+to be added.
+
+::
```

Comment:
I think `.. code-block:: python` would be better here and should enable proper code highlighting. Same for the other code block. I realize the other code blocks in this document don't do it, but still.

Reply:
Oh...except they include the `>>>` so maybe it won't work. We should standardize this and use sphinx plugins to enable/disable the `>>>` automatically.

Reply:
I changed to your suggestion and it seems to render OK.